README.md

```diff
@@ -184,6 +186,7 @@ Testing with Pyboard D, Pyboard v1.1, and TinyPico development boards
 ![pyboard d](Images/pyboardd.jpg)
 ![pyboard b](Images/pyboard.jpg)
 ![tiny pico](Images/tinypico.jpg)
+![raspberry pico](Images/raspberrypico.jpg)
```

Comment:
include `Raspberry Pi Pico`  in the line above "Testing with Pyboard D, Pyboard v1.1, and TinyPico development boards"

Reply:
Agreed; I will update the PR code accordingly.